app/frontend/src/components/designer/FormViewerActions.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <v-row class="d-print-none">
+  <v-row class="d-print-none" :class="{ 'dir-rtl': isRTL }">
     <v-col v-if="formId">
       <v-btn outlined @click="goToAllSubmissionOrDraft">
-        <span>{{ $t('trans.formViewerActions.viewAllSubmissions') }}</span>
+        <span>{{ $t('trans.formViewerActions.viewAllSubmissions') }} s </span>
```

Comment:
This extra "s" is causing problems.

<img width="221" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/c80e3461-00ad-4be1-a873-94ccc370ac13">

Reply:
I will remove it. I added that when I was trying to confirm if I'm on the right page and I forgot to remove it.